packages/truffle/scripts/test.sh

```diff
@@ -4,6 +4,11 @@ set -o errexit
 
 if [ "$GETH" == true ]; then
   mocha --timeout 50000 --grep '@ganache|@standalone' --invert --colors $@
+elif [ "$WINDOWS" == true ]; then
+# TODO: This is something we should review and go through for windows. 
+# Maybe we need some extra tests we should run for windows only.
+#  For now it is equal to GETH==true
+  mocha --timeout 50000 --grep '@ganache|@standalone' --invert --colors $@
```

Comment:
Ganache just hits more tests, since that's what `truffle test` uses under the hood by default, etc.

Of course, feel free to say "that's a separate chunk of work" and not for this PR. Definitely don't undo the work you've done for Geth!

Reply:
Ok. I will do both ganache and geth and limit test only to truffle package and contract package. 
I do it only because I just don't know what should be tested when 
* WINDOWS=true and GETH=false
* WINDOWS=true and GETH=true

You have many options in your ci.sh file. I can also test everything when GETH=false. It is more about your requirements than coding problem.